src/Twig/TwigExtension.php

```diff
@@ -75,10 +75,15 @@ public function url(string $name, $queryargs = []): string
             $query = '?' . http_build_query($queryargs);
         }
 
+        $prepend = '';
+        if ($this->pathPrefix !== null) {
+            $prepend = rtrim($this->pathPrefix, '/');
+        }
+
         // this is copy of \Slim\Slim::urlFor()
         // to mix path prefix in \Slim\Slim::urlFor
 
-        return rtrim($this->pathPrefix(), '/') . $this->router->urlFor($name) . $query;
```

Comment:
why not just use `$this->pathPrefix` here directly?

Reply:
At least in PHP8 we have struggle to concatenate null and string

Reply:
to avoid null: `$this->pathPrefix() ?: ''`